modules/signing_worker/templates/scriptworker_config.erb

```diff
@@ -0,0 +1,30 @@
+work_dir: /tmp/work
+artifact_dir: /tmp/artifacts
+taskcluster_scope_prefix: "project:releng:signing:"
+verbose: true
+local_notarization_accounts: ["account1"]
+concurrency_limit: 2
+7z: /usr/ocal/bin/7z
+mac_config:
+    dep:
+        notarize_type: multi_account
+        signing_keychain: /path/to/dep.keychain
+        supported_behaviors:
+            - mac_pkg
+            - mac_sign
+            - mac_sign_and_pkg
+            - mac_notarize
+        sign_with_entitlements: true
+        base_bundle_id: org.mozilla.firefox_dep
+        identity: ...
+        keychain_password: ...
+        pkg_cert_id: ...
+        apple_notarization_account: ...
+        apple_notarization_password: ...
+        concurrency_limit: 10
+        notarization_poll_timeout: 900
+        widevine_url: ...
+        widevine_user: ...
+        widevine_pass: ...
+        widevine_cert: ...
```

Comment:
i added `omnija_{url,user,pass}`.

Reply:
I'm not too concerned at the moment since the entire config file will need rewriting once the secrets service is available